<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Startscreen</class>
 <widget class="QDialog" name="Startscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QWigdet::Qlabel {color::white}</string>
     </property>
     <widget class="Loginform" name="page_login"/>
     <widget class="Registrform" name="page_register"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Loginform</class>
   <extends>QWidget</extends>
   <header>loginform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Registrform</class>
   <extends>QWidget</extends>
   <header>registrform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
